src/domain-engine.js

```diff
@@ -129,7 +129,7 @@ class DomainEngine {
     return next
   }
 
-  dispatch(state: Object, action: Action) {
+  dispatch(state: Object, action: Action): Object {
```

Comment:
Is this why flow discourages `get` and `set`? or am I thinking of something else? Wondering if that's related.